src/Sylius/Behat/Service/SprintfResponseEscaper.php

```diff
@@ -25,7 +25,7 @@ public static function provideMessageWithEscapedResponseContent(string $message,
             str_replace(
                 '%',
                 '%%',
-                $response->getContent()
+                json_encode(json_decode($response->getContent(), true), JSON_PRETTY_PRINT)
```

Comment:
I decided to improve JSON output shape
<img width="1440" alt="Screenshot 2021-12-15 at 18 40 00" src="https://user-images.githubusercontent.com/40125720/146390685-2c00a233-a2df-4444-9a40-a766c8af963b.png">
<img width="1440" alt="Screenshot 2021-12-15 at 18 39 07" src="https://user-images.githubusercontent.com/40125720/146390728-2850ae0d-d2ce-4721-8e6f-bf7d057244c8.png">
 to make life a little bit easier when debugging